Lib/ClassLibraryCommon/Blob/CloudBlobContainer.cs

```diff
@@ -2491,7 +2491,7 @@ internal RESTCommand<TimeSpan> BreakLeaseImpl(TimeSpan? breakPeriod, AccessCondi
             int? breakSeconds = null;
             if (breakPeriod.HasValue)
             {
-                CommonUtility.AssertInBounds("breakPeriod", breakPeriod.Value, TimeSpan.Zero, TimeSpan.MaxValue);
+                CommonUtility.AssertInBounds("breakPeriod", breakPeriod.Value, TimeSpan.FromSeconds(Constants.MinimumLeaseDuration), TimeSpan.FromSeconds(Constants.MaximumLeaseDuration));
```

Comment:
You can't have one value for both minimum lease durations since the container acquire lease minimum is 15 but the break lease min is 0